### ISSUE TITLE
I just cannot for the life of me to get it working

### ISSUE THREAD

**Issue by FinnishAviator** [ottomated/CrewLink-server#44]
I made a docker server for me and my friends, or more like tried to. I cannot for the life of me figure out what i'm doing wrong and would appreciate help. As far as i know, there is no one connected to the server. Thanks
---
![image](https://user-images.githubusercontent.com/64951407/101663072-eba32900-3a52-11eb-812f-12c126791132.png)

**Comment by Endur3D** [comment]
Yes, I also have this issue!

**Comment by crypiehef** [comment]
I set it up in Heroku and am getting the same error. The crew-link app dashboard never shows a connection either. Both are updated to the latest version.

**Comment by crypiehef** [comment]
Got it working on Heroku. Don't use the IP it shows in the dashboard.. use your URL:port in the address bar.

i.e. http://myapp.herokuapp.com:23423

**Comment by Endur3D** [comment]
> Got it working on Heroku. Don't use the IP it shows in the dashboard.. use your URL:port in the address bar.
> 
> i.e. http://myapp.herokuapp.com:23423

I can't get it to work either way, it just keeps saying "Please use another voice server..etc" Both on my own docker server and Heroku.

**Comment by Endur3D** [comment]
On further investigation, my docker container keeps crashing so that explains why I can't connect to it. It ran earlier but I still got the "Please use another voice server..."

**Comment by FinnishAviator** [comment]
Having now tried it again, i can connect to it with the localhost:9367 or whatever it is but cannot hear the others, as they have a red circle around their names

**Comment by RyqShaw** [comment]
Hey, Im having the same issue, is anyone who is using docker having the same issue?

**Comment by Endur3D** [comment]
> Hey, Im having the same issue, is anyone who is using docker having the same issue?

Yes, but right now my docker container is just crashing for some reason. I've also tried Heroku but nothing works

**Comment by Endur3D** [comment]
I got it to work with Heroku now, but all of us are red in the Crewlink app and can't hear eachother

**Comment by RyqShaw** [comment]
yeah same, there is a red circle, meaning we arent really connected, which is odd

**Comment by c-dog22** [comment]
> Having now tried it again, i can connect to it with the localhost:9367 or whatever it is but cannot hear the others, as they have a red circle around their names

Have you setup a port forward on your router that points your "externalIP":9736 to your local machine? Others will not be using the same localhost:9736 that you are using
